src/python/WMCore/WMBS/Oracle/Locations/AddPNNs.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+"""
+Oracle implementation of AddPNNs
+"""
+
+from WMCore.WMBS.MySQL.Locations.AddPNNs import AddPNNs as AddPNNsMySQL
+
+
+class AddPNNs(AddPNNsMySQL):
+    pass
```

Comment:
INSERT IGNORE doesn't work in Oracle.  should do something like below (not tested)

"""
INSERT /*+ IGNORE_ROW_ON_DUPKEY_INDEX (wmbs_pnns (id, pnn)) */  INTO wmbs_pnns (id, pnn)
 SELECT wmbs_pnns_seq.nextval, :pnn
"""